js/src/dag.js

```diff
@@ -106,7 +110,7 @@ module.exports = (common) => {
       it('returns the cid', (done) => {
         ipfs.dag.put(cborNode, {
           format: 'dag-cbor',
-          hashAlg: 'sha3-512'
+          hashAlg: 'sha2-256'
```

Comment:
Why change? This should continue to work.

Reply:
The problem is that custom hash algorithms never worked: https://github.com/ipld/js-ipld/blob/6693f89d7970002ccdc209a5c2ad5b23edf23f84/src/index.js#L233-L236 So I decided to adapt the test rather than going into the rabbit hole of fixing it.

Reply:
👍